jujugui/static/gui/src/app/components/user-profile/entity/test-entity.js

```diff
@@ -85,47 +86,27 @@ describe('UserProfileEntity', () => {
     assert.deepEqual(output, expected);
   });
 
-  it('does not render manage button for zombie models', () => {
```

Comment:
Similarly, this test is still relevant: it can be slightly changed to "does not render manage and destroy buttons for zombies".